cli/pkg/update/update.go

```diff
@@ -64,12 +65,12 @@ func LatestVersion(ctx context.Context) (string, error) {
 			return "", err
 		}
 
-		updatedAt, err := time.Parse("2006-01-02 15:04", cachedVersionUpdatedAt)
+		updatedAt, err := time.Parse(cmdutil.TSFormatLayout, cachedVersionUpdatedAt)
```

Comment:
This should not use the user-facing timestamp format, it's for internal persistance. Use `time.RFC3339` instead

Reply:
Thanks, i was checking for some better approach for this, But yeah this looks good for internal persistance :)